packages/www/src/pages/index.tsx

```diff
@@ -92,17 +92,16 @@ export default function Homepage() {
             />
             <h1>The Web Platform Framework</h1>
             <p>
-              <b>Build</b> web applications with <b>speed</b> and{' '}
-              <b>simplicity</b>
+              Build web applications with <b>speed</b> and <b>simplicity</b>
             </p>
           </hgroup>
           <div
             style={{
-              display: 'flex',
-              alignItems: 'center',
-              gap: '10px',
-              flexWrap: 'wrap',
-              justifyContent: 'center',
+              display: "flex",
```

Comment:
Should be single quote. Make sure to install biome in your IDE for keeping file consistency. This will be positive for all contributors keeping the same coding style. We, so far, are working towards respect biome standards.

Reply:
My bad! I had my usual settings set up. I suggest we add information about it to the Contributing Guide, as it doesn't say anything regarding the coding pipeline, code style or linting process. It also doesn't say anything regarding commit style either, we might want to have a GitHub workflow that runs on PRs and checks that their title conforms to conventional commits. Another option is to use husky or any other git hook tooling to help with this. Will fix the format as soon as I can.

Reply:
> I suggest we add information about it to the Contributing Guide, as it doesn't say anything regarding the coding pipeline, code style or linting process. It also doesn't say anything regarding commit style either, we might want to have a GitHub workflow that runs on PRs and checks that their title conforms to conventional commits.

Total agree with this. I'll open an issue to work this out. (In another PR).